ouroboros-consensus/test-storage/Test/Ouroboros/Storage/FS/StateMachine.hs

```diff
@@ -475,18 +481,21 @@ generator Model{..} = oneof $ concat [
     genPath :: Gen FsPath
     genPath = do
         n <- choose (0, 3)
-        mkFsPath <$> replicateM n (elements ["x", "y", "z"])
+        mkFsPath <$> replicateM n (elements ["x", "y"])
```

Comment:
Is it possible to tweak the frequencies in `genMode` instead of dropping `"z"`?

Reply:
I rolled this back. I checked again and frequencies are not affected a lot by this.